app/views/estimates/print.html.slim

```diff
@@ -44,7 +44,8 @@
     = render "evidence"
 
   - if @model.calculated?(:gross_income) || @model.calculated?(:disposable_income) || @model.calculated?(:capital)
-    h2.govuk-heading-m = t("estimates.show.#{@model.level_of_help}_how_calculated")
+    h2.govuk-heading-m style="white-space: nowrap;"
```

Comment:
FWIW this will raise a CSP error on the print preview page, as inline styles are no longer allowed.